examples/cpp-simple/Dockerfile

```diff
@@ -34,12 +34,10 @@ RUN cd cpp-simple && mkdir -p .build && \
     cmake .. -G "Unix Makefiles" -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=.bin && \
     cmake --build . --target install
 
-FROM debian:stretch
+FROM debian:buster
 RUN useradd -u 1000 -m server
 
-COPY --from=builder /project/cpp-simple/.build/.bin/cpp-simple /home/server/cpp-simple
-RUN chown -R server /home/server && \
-    chmod o+x /home/server/cpp-simple
+COPY --from=builder --chown=server:server /project/cpp-simple/.build/.bin/cpp-simple /home/server/cpp-simple
```

Comment:
You removed the `chmod o+x` line. Was it intended?

Reply:
I have tested the resulted file, and it has the same permissions. So this two strings should be equivalent.